### DISCUSSION TITLE
Legacy ASP.NET Framework 4.8 MVC app

### DISCUSSION THREAD

**Discussion by 1jmj** [Hawxy/Clerk.Net#11] [Q&A] [answered]
Is this just an API wrapper or can it be used for auth in a MVC website, and be compatible with the build in AuthorizeAttribute

**Comment by Hawxy** [comment, marked as answer]
This is just an API wrapper. Clerk doesn't currently support OIDC so setting it up in MVC is a bit annoying. You might be able to hack something together with `Microsoft.Owin.Security.OAuth` + https://clerk.com/docs/advanced-usage/clerk-idp
